pkg/api/vlabs/validate.go

```diff
@@ -1176,7 +1176,7 @@ func (a *Properties) validateWindowsProfile() error {
 			false,
```

Comment:
Should we instead figure out how to plumb in isUpdate here instead of hard-coding the false? 
I think that is the initial cause of the issues.

Reply:
(But if that is a lot of work I'm fine with these changes)

Reply:
Oh, maybe you're right that that's the way to do this! The current UT in this PR should validate if it's as easy as that.

Reply:
Nice - I think I broke this when i was refactoring windows validation in a previous PR :(